DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -751,6 +752,111 @@ internal async Task UpdateUserVoiceStateAsync(ulong guild_id, ulong user_id, ulo
         }
         #endregion
 
+        #region Stickers
+
+        internal async Task<DiscordMessageSticker> GetGuildStickerAsync(ulong sticker_id, ulong guild_id)
+        {
+            var route = $"{Endpoints.GUILDS}/:guild_id{Endpoints.STICKERS}/:sticker_id";
+            var bucket = this.Rest.GetBucket(RestRequestMethod.GET, route, new {guild_id, sticker_id}, out var path);
+            var url = Utilities.GetApiUriFor(path);
+
+            var res = await this.DoRequestAsync(this.Discord, bucket, url, RestRequestMethod.GET, route).ConfigureAwait(false);
+            var ret = JObject.Parse(res.Response).ToDiscordObject<DiscordMessageSticker>();
+
+            ret.Discord = this.Discord;
+            return ret;
+        }
+
+        internal async Task<DiscordMessageSticker> GetStickerAsync(ulong sticker_id)
+        {
+            var route = $"{Endpoints.STICKERS}/:sticker_id";
+            var bucket = this.Rest.GetBucket(RestRequestMethod.GET, route, new {sticker_id}, out var path);
+            var url = Utilities.GetApiUriFor(path);
+
+            var res = await this.DoRequestAsync(this.Discord, bucket, url, RestRequestMethod.GET, route).ConfigureAwait(false);
+            var ret = JObject.Parse(res.Response).ToDiscordObject<DiscordMessageSticker>();
+
+            ret.Discord = this.Discord;
+            return ret;
+        }
+
+        internal async Task<IReadOnlyList<DiscordMessageSticker>> GetStickersAsync(ulong guild_id)
+        {
+            var route = $"{Endpoints.GUILDS}/:guild_id{Endpoints.STICKERS}";
+            var bucket = this.Rest.GetBucket(RestRequestMethod.GET, route, new {guild_id}, out var path);
+            var url = Utilities.GetApiUriFor(path);
+
+            var res = await this.DoRequestAsync(this.Discord, bucket, url, RestRequestMethod.GET, route).ConfigureAwait(false);
+            var ret = JObject.Parse(res.Response).ToDiscordObject<IEnumerable<DiscordMessageSticker>>();
```

Comment:
stickers: Error reading JObject from JsonReader. Current JsonReader item is not an object: StartArray. Path '', line 1, position 1.

👀

Reply:
```json
[
    {
        "id": "859914667031789578",
        "name": "Mike",
        "tags": "llama",
        "type": 2,
        "format_type": 1,
        "description": "Mike Stickeeeeeeeeeer",
        "asset": "",
        "available": true,
        "guild_id": "858089281214087179",
        "user": {
            "id": "856780995629154305",
            "username": "Catra~Chan",
            "avatar": "5883e678510c8f46ba61ce2c8d4a3b22",
            "discriminator": "0001",
            "public_flags": 64
        }
    },
    {
        "id": "859934445338624001",
        "name": "Miku Dance",
        "tags": "star_struck",
        "type": 2,
        "format_type": 2,
        "description": "Dancing cute miku",
        "asset": "",
        "available": true,
        "guild_id": "858089281214087179",
        "user": {
            "id": "856780995629154305",
            "username": "Catra~Chan",
            "avatar": "5883e678510c8f46ba61ce2c8d4a3b22",
            "discriminator": "0001",
            "public_flags": 64
        }
    }
]
```

Reply:
It should be JArray, I'm aware.